src/urg_node.cpp

```diff
@@ -38,10 +38,9 @@ int main(int argc, char **argv)
   // Initialize node and nodehandles
   rclcpp::init(argc, argv);
   auto node = std::make_shared<urg_node::UrgNode>("urg_node");
-  node->initSetup();
 
   // Run the urg node
-  node->run();
+  //node->run();
```

Comment:
Accidently commented ? I couldn't find run() is not getting called anywhere else

Reply:
Same comment as below. Most likely by accident.